src/extension.ts

```diff
@@ -72,6 +71,7 @@ export async function activate(context: vscode.ExtensionContext) {
 	applyUserSettings();
 
 	mainOutputChannel = vscode.window.createOutputChannel("Apache Camel K");
+	await installAllTutorials(context);
```

Comment:
moved installation of tutorials at the beginning of the activation. it allows to have Tutorials displayed faster on first start as it doesn't have to wait that the kamel binary is downloaded

Reply:
the question for me is "Can you work with the tutorial without having the download of kamel binary finished?". I mean you can surely read the tutorial but you cannot execute any `kamel` commands.

Reply:
the commands should wait for the end of the download. if it is not the case, it is a bug in the command that can be hit outside of the context of the tutorial anyway